src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/GUID.cs

```diff
@@ -40,7 +40,7 @@ internal sealed class GUIDPureFunction : BuiltinFunction
 
         public override bool IsSelfContained => true;
 
-        public override bool SupportsParamCoercion => false;
+        public override bool SupportsParamCoercion => true;
```

Comment:
>true [](http://example.com/codeflow?start=54&length=4)

Is this a potential breaking change? (last time we did something similar we broke tests in paclient)

Reply:
This is a conversation we should have more broadly, pulled out of this PR.